pom.xml

```diff
@@ -43,7 +43,7 @@
     <jaxb-impl.version>2.3.6</jaxb-impl.version>
     <jaxb-core.version>4.0.0</jaxb-core.version>
     <mod-configuration-client.version>5.9.2</mod-configuration-client.version>
-    <edge.common.version>4.6.0</edge.common.version>
+    <edge.common.version>4.7.0</edge.common.version>
     <vertx-stack-depchain.version>4.5.4</vertx-stack-depchain.version>
```

Comment:
We also should include this security vulnerability fix https://github.com/folio-org/edge-oai-pmh/pull/117
and please update News with this story